source/server-discovery-and-monitoring/tests/errors/prefer-error-code.yml

```diff
@@ -0,0 +1,53 @@
+description: Do not check errmsg when code exists
```

Comment:
This is the one test that ensure that errmsg strings are no longer checked when the code field exists.